liquidity-sources/Cargo.toml

```diff
@@ -27,7 +27,7 @@ prometheus-metric-storage = "0.4"
 reqwest = { version = "0.11", features = ["json"] }
 scopeguard = "1.1.0"
 serde = "1.0"
-serde_json = "1.0"
+serde_json = { version = "1.0", features = ["preserve_order"]}
```

Comment:
Is the feature really needed?

Reply:
Hm, not sure if it is a must to work, but it is necessary for UT to pass (since serialization/deserialization of BTreeMap does not preserve order otherwise), and also would be nice to have the same order for later debugging/comparison with lpbook results.

Reply:
I think the order is already deterministic just by using BTreeMap (instead of HashMap). It should be ordered by the PartialOrd implementation of the key (string). So if it doesn't pass the tests yet I think it should be possible by reording the fields in the `json!` macro in the tests to have the same order.

Reply:
I'd prefer that because by using the preserve_order feature we force all crates that use this crate and serde_json to opt into the ordering preservation features (because of how cargo works) which is less efficient.

Reply:
Makes sense. Removed `preserve-order` feature.